templates/board.html

```diff
@@ -68,7 +69,7 @@
                                 <div class="articles feed-articles">
                                     {% for article in articles %}
                                         <div class="article {% if article.is_fresh %}is-article-fresh{% endif %}">
-                                            <div class="article-title">{{ article.icon|safe }}<a href="{{ article.url }}" target="_blank">{{ article.title|striptags }}</a></div>
+                                            <div class="article-title">{{ article.icon|safe }}<a href="{{ article.url }}" target="_blank">{{ article.title|striptags|bleach }}</a></div>
```

Comment:
Do we need the `striptags` filter then?

Reply:
We can use this setting, rm striptags
BLEACH_STRIP_TAGS = True

[https://django-bleach.readthedocs.io/en/latest/settings.html#settings](Doc)

done :)

Reply:
I though it’s done automatically, heh. Ok👌